database/info.txt

```diff
@@ -1 +0,0 @@
-This folder contains the data
```

Comment:
With this change the database/ folder is no more created automatically on a fresh clone, is this safe?